docs/get-started/getting-started-alluxio.md

```diff
@@ -0,0 +1,200 @@
+---
+layout: page
+title: Alluxio
+nav_order: 5
+parent: Getting-Started
+---
+# Get Started with RAPIDS with Alluxio
+
+RAPIDS plugin can remarkably accelerate the computing part of the whole SQL query by
+leveraging GPUs, but it’s hard to accelerate the data reading process when the data
+is in the cloud file system because of network overhead. Alluxio is an open source
+data orchestration platform that brings your data closer to compute across clusters,
+regions, clouds, and countries for reducing the network overhead. This guide will
+go through how to set up the RAPIDS Accelerator for Apache Spark with Alluxio with
+a on premise cluster.
+
+## Prerequisites
+
+Assuming that user has successfully setup and ran the RAPIDS Accelerator with on premise
+cluster according to [this doc](getting-started-on-prem.md)
+
+This guide will deploy Alluxio on Yarn cluster with 2 NodeManagers and 1 ResourceManager,
+and describe the instructions to configure Swiftstack as Alluxio’s under storage system.
```

Comment:
```suggestion
This guide assumes the user has successfully setup and run the RAPIDS Accelerator in an on-premise
cluster according to [this doc](getting-started-on-prem.md). 

This guide will go through deployment of Alluxio in a Yarn cluster with 2 NodeManagers and 1 ResourceManager.  It will describe how to configure an S3 compatible filesystem as Alluxio’s underlying 
storage system.
```